src/main/resources/lib/ssb/municipalityAlert.es6

```diff
@@ -17,7 +17,7 @@ export const get = (key) => {
 export const list = ( municipalCode ) => {
   const now = new Date()
   return query({
-    query: `data.municipalCodes IN ('${municipalCode}') AND publish.from LIKE '*' AND (publish.to NOT LIKE '*' OR publish.to > '${now.toISOString()}')`,
+    query: `data.selectAllMunicipals = 'true' OR data.municipalCodes IN ('${municipalCode}') AND publish.from LIKE '*' AND (publish.to NOT LIKE '*' OR publish.to > '${now.toISOString()}')`,
```

Comment:
Denne linjen blir for lang. Lintingen lager error.

Reply:
Sånn, delt den opp nå :D